### PR TITLE
Alias the EXTEND register to match reference manual

### PR DESCRIPTION
**Pull request by cnlohr**

### CODE REVIEW

File: ch32v003fun/ch32v003fun.h [hunk 1]
```diff
@@ -1222,6 +1222,13 @@ typedef struct
     __IO uint32_t EXTEN_CTR;
 } EXTEN_TypeDef;
 
+/* The reference manual for the 32v2xx/v3xx refernce this as "CTR" on "EXTEND" so adding an alias here. for any processor. */
```
Comment (TommyMurphyTM1234) [inline ch32v003fun/ch32v003fun.h:1225]: Typos:
- "refernce" should be ”reference"
- "on" should be "or"?
- "32v2xx" should be "ch32v2xx"?
- ". for any processor" doesn't seem to read properly?

File: ch32v003fun/ch32v003fun.h [hunk 2]
```diff
@@ -2757,6 +2764,7 @@ typedef struct
 #define OB                                      ((OB_TypeDef *)OB_BASE)
 #define ESIG                                    ((ESG_TypeDef *)ESIG_BASE)
 #define EXTEN                                   ((EXTEN_TypeDef *)EXTEN_BASE)
+#define EXTEND                                  ((EXTEND_TypeDef *)EXTEN_BASE)  // Alias to EXTEN
```
Comment (TommyMurphyTM1234) [inline ch32v003fun/ch32v003fun.h:2767]: Typo
- `EXTEN` should be `EXTEND`

**Review by TommyMurphyTM1234 — COMMENTED** [review]
Some small comments which I hope are useful.

File: ch32v003fun/ch32v003fun.h (hunk 2 above)
Comment (cnlohr) [reply ch32v003fun/ch32v003fun.h:2767]: No, this is what I intended to do.  I am aliasing `EXTEND` to `EXTEN`, but with different register names.

File: ch32v003fun/ch32v003fun.h (hunk 1 above)
Comment (cnlohr) [reply ch32v003fun/ch32v003fun.h]: Fixed in new rev.  Thanks for the feedback.  I will wait for it to pass CI.

**Pull request merged**